requirements.txt

```diff
@@ -1,2 +1,3 @@
 Flask==1.1.1
 python-dotenv==0.12.0
+requests
```

Comment:
You should try to include versions, because you normally want to control updating to newer versions.

After e.g. `pip install requests` you can run `pip freeze` to view the installed dependencies with version numbers. All the extra ones not included in requirements.txt are sub-dependencies, i.e. dependencies of your dependencies.